common/src/main/java/com/viaversion/viaversion/protocols/v1_12_2to1_13/data/ParticleRewriter.java

```diff
@@ -96,7 +95,7 @@ public class ParticleRewriter {
 
     public static Particle rewriteParticle(int particleId, Integer[] data) {
         if (particleId >= particles.size()) {
-            Via.getPlatform().getLogger().severe("Failed to transform particles with id " + particleId + " and data " + Arrays.toString(data));
+            Protocol1_12_2To1_13.LOGGER.severe("Failed to transform particles with id " + particleId + " and data " + Arrays.toString(data));
```

Comment:
You can remove this error and just ignore it

Reply:
Is there any specific reason for this except not having a static LOGGER instance? (Which we now need anyway in this protocol)